examples/src/scala/org/pantsbuild/example/BUILD

```diff
@@ -1,26 +1,6 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-scala_library(
-  name = 'jvm-run-example-lib',
```

Comment:
This is moved to a subdirectory. Keeping it at the root has major problems that we then need to have the IT pull in dependencies for other targets in this BUILD which have nothing to do with the IT.